src/main/java/gr/grnet/pccapi/dto/DomainDto.java

```diff
@@ -1,12 +1,39 @@
 package gr.grnet.pccapi.dto;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
+@Schema(name="DomainResponse", description="An object represents the stored Scientific Domain.")
```

Comment:
Scientific Domain is the branch of science, scientific discipline that is related to the project using the Prefix. It uses the Classification Information/Scientific Domain from EOSC Profiles .